package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nebulr-group/nblocks-react",
-  "version": "2.3.2",
+  "version": "2.3.3",
```

Comment:
I would argue this is a feature addition, thus `2.4.0` would be a better version. But not blocking this time!

Reply:
Edit: since you change some method calls called by external code a major version name would be even better.